softdevice_controller/include/sdc.h

```diff
@@ -311,7 +311,6 @@ int32_t sdc_support_adv(void);
  *
  * After this API is called, the controller will support the HCI commands
  * and events related to the slave role.
- * This state requires the advertising state to be supported.
```

Comment:
Seems unrelated, why is this changed?

Reply:
The SoftDevice Controller automatically supports the advertising state if the application enables slave support